db/migrate/20240314230024_create_assignments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAssignments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :assignments do |t|
+      t.references :platform, foreign_key: true
+      t.string :assignment_name
```

Comment:
```suggestion
      t.string :name
```
Since the table is assignments, having `assignment` in the column is redundant.